MapboxNavigationObjc/include/MapboxNavigation.h

```diff
@@ -7,4 +7,4 @@ FOUNDATION_EXPORT double MapboxNavigationVersionNumber;
 FOUNDATION_EXPORT const unsigned char MapboxNavigationVersionString[];
 
 #import "../MBRouteVoiceController.h"
-#import "../MLNMapView+MLNNavigationAdditions.h"
+#import <MapLibre/Mapbox.h>
```

Comment:
While we are at it, can we rename this file to `MapLibre.h` or is this in another package?

Reply:
It's not in this package.